torch/_subclasses/functional_tensor.py

```diff
@@ -29,8 +29,36 @@ class FunctionalTensor(torch.Tensor):
     # this is an "infra" mode with lower dispatching precedence.
     _mode_key = torch._C._TorchDispatchModeKey.FUNCTIONAL
 
+    # Note: FunctionalTensor is not quite like the functorch wrapper subclasses,
+    # because today functionalization runs *below* the conj/neg/zero dispatch keys.
+    # We could consider changing this to make it more inline with functorch.
+    _extra_dispatch_keys = torch._C._additional_keys_to_prop_for_wrapper_tensors.add(
+        torch._C.DispatchKey.ZeroTensor
+    )
```

Comment:
nit: The comment doesn't really explain what we're doing here.

Reply:
Fair, I'll make this clearer